_data/speakers.yml

```diff
@@ -211,17 +211,21 @@
   position: Professor at University of Toronto, Chair in Biomedical Informatics and Artificial Intelligence at the Hospital for Sick Children
   bio: "Anna Goldenberg is famous for her research on how machine learning can be used to map the heterogeneity seen in various human diseases. Her lab develops methodologies to identify patterns in collected data and improve patient outcomes. The “Similarity Network Fusion” she co-developed was the first patient data integration method that improved survival outcome predictions in different cancers.
 
-  Goldenberg is professor at the University of Toronto and holds a CIFAR AI Chair. She was the first to receive a chair in biomedical informatics and AI at the Hospital for Sick Children.
+  Goldenberg is professor in the departments of Computer Science and the Laboratory Medicine and Pathobiology at the University of Toronto. She is a Varma Family Chair in Biomedical Informatics and Artificial Intelligence at SickKids Research Institute as well as a CIFAR AI chair at  the Vector Institute.
+
+  Dr. Goldenberg trained in machine learning at Carnegie Mellon University, with a postdoctoral focus in computational biology and medicine. The current focus of her lab is on developing and deploying machine learning models to healthcare.
 
   Among other awards, Anna Goldenberg received the Early Researcher Award from the Ministry of Research and Innovation.
 
-  Since she is strongly committed to creating responsible AI to benefit patients across a variety of conditions, she also put forward publication such as “Do no harm: a roadmap for responsible machine learning for health care” (Nature Medicine, 2019).
+  Dr. Goldenberg’s lab is strongly committed to creating responsible AI to benefit patients across a variety of conditions.
   "
   photo: anna_goldenberg.jpg
   alttext: Prof. Anna Goldenberg
-  session: tbd
+  title: "Time series ML for deployment in healthcare"
+  abstract: "There is growing support and excitement around AI integration into clinical practice. And yet, enabling AI in healthcare broadly is replete with many obstacles including fundamentally unresolved machine learning issues. In this talk I will cover several of our contributions to time series modeling including explainability, representation learning and generative modeling that we have worked on to date. I will also discuss some of the broader computer science contributions we had to make in order to get us closer to deployment. Finally, if time permits, I will touch on an important question of feedback loop, i.e. what happens when an ML solution is deployed and is starting to affect the labels that we are using to retrain and improve our models."
+  session: inperson
   date: 20221121
```

Comment:
If it is i person, the date should be 20221208, right? I believe it doesn't matter, but just in case.